indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -571,6 +574,26 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_avg_per_question_per_track(self):
+        ratings = {}
+        abstract_ratings = (AbstractReviewRating.query
+                            .join(AbstractReviewRating.review)
+                            .join(AbstractReviewRating.question)
+                            .options(contains_eager('review'), contains_eager('question'))
+                            .filter(AbstractReview.abstract == self,
+                                    ~AbstractReviewQuestion.is_deleted)
+                            .all())
+        for rating in abstract_ratings:
+            if rating.question.no_score:
+                continue
+            ratings.setdefault(rating.review.track_id, {})
```

Comment:
I think it makes more sense to not include the tracks with no reviews yet. So no 0 avg.